modules/globebrowsing/src/renderableglobe.cpp

```diff
@@ -1606,7 +1614,7 @@ void RenderableGlobe::setCommonUniforms(ghoul::opengl::ProgramObject& programObj
         // If it does not it will still produce "correct" normals. If the resolution is
         // higher the shadows will be softer, if it is lower, pixels will be visible.
         // Since default is 64 this will most likely work fine.
-        constexpr float TileDelta = 1.f / DefaultSkirtedGridSegments;
+        constexpr float TileDelta = 1.f / 512;
```

Comment:
Should this stay unchanged?

Reply:
since [the default texture size](https://github.com/OpenSpace/OpenSpace/blob/6d87eb39b1e78149e57ed968162a84d61f19eede/modules/globebrowsing/src/tiletextureinitdata.cpp#L109) for heightmaps changed I believe it makes sense to change this value to match. It is also conceptually weird that the resolution used for normal calculation is based on how many segments the skirt of a tile has.  I can't really see any major visual difference between leaving it or changing it but for consistency I think it should be changed.
But the comment should be updated though! (if we decide to change this value that is)

Reply:
I'm fine with the change in resolution, but I think the `DefaultSkirtedGridSegments` variable should be defined instead.

Reply:
Makes sense, lets define a new variable then to hold this value? 
DefaultSkirtedGridSegments_ is also used to determine the resolution of the grid and it feels like an unnecessary performance degradation to increase the number of vertices on a grid by a factor 64 😄

Reply:
Yea, sounds good!